website/content/docs/drivers/docker.mdx

```diff
@@ -239,10 +239,12 @@ config {
   configuration on the host (which is outside the scope of Nomad). Valid values
   pre-docker 1.9 are `default`, `bridge`, `host`, `none`, or `container:name`.
 
-  The default `network_mode` for tasks that use [Connect] will be
-  `container:<name>`, where the name is the container name of the parent
-  container used to share network namespaces between tasks. You should not set
-  `network_mode` for Connect-enabled tasks.
+  The default `network_mode` for tasks that use group networking in [`bridge`]
+  mode will be `container:<name>`, where the name is the container name of the
+  parent container used to share network namespaces between tasks. If you set
+  the `network_mode` field in this case, the container will be unable to reach
+  other containers in the task group. This will prevent [Connect]-enabled
+  tasks from reaching the Envoy sidecar proxy. See [GH-10725].
```

Comment:
Not obvious to me the value of linking to the issue here. It doesn't seem to provide more info on why setting `network_mode` is a bad idea here, and seems to raise unanswered questions so far. If want feedback from customers, maybe we can make an explicit call to action.

Reply:
You're right, that's not helpful. I'll tweak the language a bit here to me more firm about why and then leave it at that.